source/parse.h

```diff
@@ -3010,6 +3025,12 @@ class parser
                     next();
                     return {};
                 }
+                if ( // check if unnamed function ends with semicolon
```

Comment:
Thanks! I'm mainly going to suggest comment/message rewordings please...

"check if a single-expression function is followed by an extra second semicolon"

Reply:
working on it...